md/release-notes.md

```diff
@@ -1,18 +1,143 @@
+
+### Add a display name and display description to a page, form, or layout from the Editor
+Display name and description are used when the page, form, or layout are imported in the portal's Resources page.
+They could be edited by opening the page.properties file. Now, they can be directly edited in the page, form, or layout editor of the UI Designer, by clicking on the new "Tag" button.
+
+<a id="bonita-calls"/>
+
+### Get a summary of the calls to Bonita REST API made in a page, form, or layout
+Also by clicking on the "Tag" button, you can get a summary of all REST API calls made by the current artifact to the Bonita REST API.
+This helps validating if all expected calls are well implemented in the artifact in its current state.
+
+<a id="bonita-theme"/>
+
+### New Living Applications theme
+A new theme is available for any Living Application you create: "bonita-default-theme". 
+
+<a id="operations-apiaccessor"/>
+
+### Warning on best practices when using API in operations
+In diagram operations, the right operand should only use the apiAccessor with read-only methods. After helping several customers with scripts calling the apiAccessor with "write"-type methods, we have created a warning in the studio, when detecting the apiAccessor is called in an opreation script.
 
 <a id="technology-updates"/>
 
 ## Technology updates
 
+<a id="kerberos"/>
+
+### Integration of SSO Kerberos (Teamwork, Efficiency, Performance edition)
+Bonita works with Single-Sign-On (SSO) solutions using Kerberos protocol:
+- Use standard authentication service
+- Connect to Bonita Living Applications with company credentials (per tenant configuration)
+- Works for Windows and many UNIX and UNIX-like operating systems: FreeBSD, Apple's MacOS, RED Hat Enterprise Linux, Oracle's Solaris..
+
+<a id="other-dependencies"/>
+
+## Other dependency updates
+
+<a id="spring"/>
+
+### Spring
+Some internal libraries have been updated to newer versions:
+* spring framework version is now 5.0.10.RELEASE
+* spring-boot version is now 2.0.6.RELEASE
+
+<a id="tomcat"/>
+
+### Tomcat 8.5.34
+Update to a newer Tomcat version.
+
 <a id="feature-removals"/>
 
 ## Feature removals
 
+<a id="gwt"/>
+
+### GWT technology in the Portal (1/n)
+As Google Web Toolkit is an ageing technology, Bonita needs to transform the pages of Bonita Portal step by step.
+In this Bonita 7.8 release, two pages have been redone: 
+ - Process list for the profile User
+   It has been done using React. It is made available in the portal's Resources page, so it can be used in any application. It cannot be edited in the UI Designer.
+ - Tenant status page for the profile Technical Administrator
+   It has been done using the UI Designer and the new modal container. Since it is dedicated to a profile critical for the good initialization and management of Bonita, the page has not been made available in the portal's resources. It cannot be added to any application nor be edited in the UI Designer.
```

Comment:
"It can neither be used in any application nor exported to be edited in the UI Designer" ?

Reply:
Checking with Mickey and implementing what she answers back.